### PR TITLE
[SDPA-3650] Added nginx conf to set header.

### PR DESCRIPTION
**Pull request by GROwen**
# Checklist
- [x] Affected Issues have been mentioned in the Closing issues section
- [ ] Documentation has been written/updated.
- [ x ] Changelog entry has been written

Explain the **details** for making this change. What existing problem does the pull request solve?

# Changelog Entry

Feature - Added new header `X-Sdp-App-Type` to Nginx config.

### CODE REVIEW

File: bay/images/nginx/helpers/200_x-sdp-app-type-header.conf [hunk 1]
```diff
@@ -0,0 +1 @@
+add_header X-Sdp-App-Type "tide";
```
Comment (steveworley) [inline bay/images/nginx/helpers/200_x-sdp-app-type-header.conf]: We might be able to do this with environment variable substitution. The base Amazee nginx container runs `ep` over the helpers directory during the entry of the container.

`ep` (envplate) supports environment variable substitution in static files.

```suggestion
add_header X-Sdp-App-Type "${SDP_APP_TYPE:-tide}";
```

Comment (steveworley) [reply bay/images/nginx/helpers/200_x-sdp-app-type-header.conf]: https://github.com/amazeeio/lagoon/blob/master/images/nginx/docker-entrypoint#L22

File: bay/images/nginx/helpers/200_x-sdp-app-type-header.conf [hunk 2]
```diff
@@ -0,0 +1 @@
+add_header X-Sdp-App-Type "${SDP_APP_TYPE:-tide}";
```
Comment (kurtfoster) [inline bay/images/nginx/helpers/200_x-sdp-app-type-header.conf]:
```suggestion
add_header x-sdp-app-type "${SDP_APP_TYPE:-tide}";
```

Comment (GROwen) [reply bay/images/nginx/helpers/200_x-sdp-app-type-header.conf]: @kurtfoster What's the argument to set it as lowercase? I followed the naming convention used for standards headers (and the way other projects i.e. GovCMS set them) and Tim noticed that Section.io transforms these to lowercase when the CDN is used.

Comment (GROwen) [reply bay/images/nginx/helpers/200_x-sdp-app-type-header.conf]: As discussed @kurtfoster we'll keep this with the current case structure to maintain the naming convention commonly used for HTTP headers. They're case insensitive so it doesn't matter either way - it's merely convention.

**Pull request merged**